src/components/Preferences/Preferences.tsx

```diff
@@ -104,6 +125,16 @@ export function Preferences() {
             <ResourcesValidationSettings />
           </div>
         </VerticalTabs.Content>
+        <VerticalTabs.Content id={3}>
+          <div>
+            {isExtensibilityEnabled && (
```

Comment:
probably this `isExtensibilityEnabled` should be higher before `<VerticalTabs.Content id={3}>`